### PR TITLE
Add end to end tests to the renameTeamCmdF mmctl command

### PR DESCRIPTION
**Pull request by gupsho**
#### Summary
Add end to end tests to the renameTeamCmdF mmctl command

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/15674

### CODE REVIEW

File: commands/team_e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+func (s *MmctlE2ETestSuite) TestRenameTeamCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Error renaming team which does not exist", func(c client.Client) {
+		printer.Clean()
+		oldTeamName := "existingName"
+		cmd := &cobra.Command{}
+		args := []string{""}
+		args[0] = oldTeamName
```
Comment (streamer45) [inline commands/team_e2e_test.go:21]: I'd probably name this `nonExistentTeamName` to make it more clear.
Also we can compress a bit the initialization:

```
args := []string{"nonExistentTeamName"}
```

File: commands/team_e2e_test.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		err := renameTeamCmdF(c, cmd, args)
+		s.Require().EqualError(err, "Unable to find team 'existingName', to see the all teams try 'team list' command")
+	})
+
+	s.RunForSystemAdminAndLocal("Rename a valid team", func(c client.Client) {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		args := []string{""}
+		args[0] = s.th.BasicTeam.Name
```
Comment (streamer45) [inline commands/team_e2e_test.go:33]: Same here, no real need for two assignments.

**Review by streamer45 — COMMENTED** [review]
Thanks @gupsho! Looks good overall :+1: 

I've added a couple of minor suggestions. Also, I'd probably add an additional test to check for a permissions error to be returned if the user attempts to rename an existing team.

**Comment by gupsho** [comment]
> Thanks @gupsho! Looks good overall 👍
> 
> I've added a couple of minor suggestions. Also, I'd probably add an additional test to check for a permissions error to be returned if the user attempts to rename an existing team.

Done.

**Review by mgdelacroix — APPROVED** [review]
LGTM! Thanks a lot for the contribution @gupsho!! :confetti_ball:

File: commands/team_e2e_test.go (hunk 2 above)
Comment (gupsho) [reply commands/team_e2e_test.go]: Done.

**Review by streamer45 — APPROVED** [review]

**Review by lindy65 — APPROVED** [review]
Thanks @mgdelacroix - looks good to me 👍

**Pull request merged**